<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowUI</class>
 <widget class="QWidget" name="WindowUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FindMinutiae</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="TabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TabDB">
      <attribute name="title">
       <string>DB Creation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="DBFilesHorLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="GUI::LineEditDropReceiver" name="DBFilesLineEdit">
           <property name="placeholderText">
            <string>Path to folder with audio files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DBFilesBrowseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="DBPlotNameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="DBPlotName">
           <property name="placeholderText">
            <string>Name of the plot</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="DBCreateButton">
         <property name="text">
          <string>Create Database</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GUI::DatabaseChart" name="DBChartView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="PlotCtrlHorLayout">
         <item>
          <widget class="QPushButton" name="PlotCtrlCleanButton">
           <property name="text">
            <string>Clean Plot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DBSaveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DBLoadButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabMatching">
      <attribute name="title">
       <string>Matching</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="MIntraCurveHLayout">
         <item>
          <widget class="QLabel" name="MIntraCurveLabel">
           <property name="text">
            <string>Suspect curve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="MIntraCurveComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentText">
            <string notr="true"/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="MExtraCurveHLayout">
         <item>
          <widget class="QLabel" name="MExtraCurveLabel">
           <property name="text">
            <string>Extra-speaker curve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="MExtraCurveComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="MUnknownHorLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="MULabel">
           <property name="text">
            <string>Unknown Voice</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="GUI::LineEditDropReceiver" name="MULineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Folder with unknown voice</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MUBrowseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="MatchingResultsLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Distribution Similarity Estimation</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GUI::DatabaseChart" name="MatchingChartView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="MatchingCtrlHorLayout">
         <item>
          <widget class="QPushButton" name="MCtrlMatchButton">
           <property name="text">
            <string>Match</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MCtrlResetButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MLoadDatabaseButton">
           <property name="text">
            <string>Load Database</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="FileInspectorTab">
      <attribute name="title">
       <string>File Inspector</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="FIFileLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item row="0" column="0">
          <widget class="GUI::LineEditDropReceiver" name="FILineEdit">
           <property name="placeholderText">
            <string>Path to an audio file...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="FIBrowseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="GUI::ChartRecWidget" name="ChartShowRec" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="HorRecordControls">
         <item>
          <widget class="QPushButton" name="ButtonPreviousRecord">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Resources/48x48/Prev.png</normaloff>../../Resources/48x48/Prev.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LabelRecordDescription">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Record Description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonNextRecord">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Resources/48x48/Successive.png</normaloff>../../Resources/48x48/Successive.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="GUI::ChartRecWidget" name="ChartShowRecSpectrum" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="CheckLogScale">
         <property name="text">
          <string>Log scale</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SettingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QScrollArea" name="SettingsScroll">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="SettingsScrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>58</width>
            <height>16</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="ResetParametersButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUI::LineEditDropReceiver</class>
   <extends>QLineEdit</extends>
   <header location="global">GUI/LineEditDropReceiver.hpp</header>
  </customwidget>
  <customwidget>
   <class>GUI::ChartRecWidget</class>
   <extends>QWidget</extends>
   <header location="global">GUI/DatabaseChart.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GUI::DatabaseChart</class>
   <extends>QWidget</extends>
   <header location="global">GUI/DatabaseChart.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
